pkgs/applications/science/logic/cadical/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "cadical";
-  version = "1.2.1";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner = "arminbiere";
     repo = "cadical";
     rev = "rel-${version}";
-    sha256 = "1a66xkw42ad330fvw8i0sawrmg913m8wrq5c85lw5qandkwvxdi6";
+    sha256 = "05lvnvapjawgkky38xknb9lgaliiwan4kggmb9yggl4ifpjrh8qf";
   };
 
   dontAddPrefix = true;
```

```suggestion
  doCheck = true;
  dontAddPrefix = true;

```

Running tests seems to be pretty fast, maybe you can enable them so that any future regressions can be caught?